files/run_worker.sh

```diff
@@ -16,6 +16,12 @@ else
   LOGLEVEL="DEBUG"
 fi
 
+if [[ ${HOSTNAME} == "packit-worker-1" ]]; then
+  QUEUES="short-running"
+else
+  QUEUES="short-running,long-running"
+fi
```

Comment:
What about stage deployment with only one worker?

Reply:
Aha, we are indexing from `0`..;)

Reply:
Maybe not necessary but maybe, we can have an env. variable with the number of short-running-queue workers so we can edit the proportion more easily. But this will be more relevant once we have a different image for different types of workers and that can be done on the openshift level and not via env. var.

Reply:
not so nice solution could be to have env vars e.g. `SHORT_RUNNING`=x, `LONG_RUNNING`=y (so currently we would have `SHORT_RUNNING`=1, `LONG_RUNNING`=0), `packit-worker-0` would always serve both queues and then `x` workers beginning from `packit-worker-1` would serve the short running queue and `y` workers beginning from `packit-worker-{x+1}` would serve only long-running queue (in this way we would be able to set `QUEUES` in the `run_worker` script depending on the `HOSTNAME`)...is this something that you had in mind? we would need to make sure that `SHORT_RUNNING`+`LONG_RUNNING`+1 = number of workers, or schedule the number of workers according to this calculation

Reply:
@lbarcziova Yes, that's what I had in mind.

But, I am not sure if this is needed for now:

* If we want to differentiate images for different types of workers (what I think we've discussed), we can leave that work for later. (Scaling will be done on openshift level, not worker code level.)
* If we don't have such a change in mind,  having `SHORT_RUNNING`/`LONG_RUNNING` env. variables will be nice.

Reply:
I implemented it in the way described in the comment, PTAL